packages/ckeditor5-engine/tests/model/markercollection.js

```diff
@@ -53,7 +53,14 @@ describe( 'MarkerCollection', () => {
 			expect( marker.managedUsingOperations ).to.be.false;
 			expect( marker.affectsData ).to.be.false;
 			expect( marker.getRange().isEqual( range ) ).to.be.true;
-			sinon.assert.calledWithExactly( markers.fire, 'update:name', result, null, range );
+
+			expect( markers.fire.firstCall.args ).to.deep.equal( [
```

Comment:
I tried to write a test with `sinon.assert` +  `sinon.match()` but `sinon.match()` has a bug when an object has circular refs.